### PR TITLE
[MTB] refactored code and added other ns flag

### LINKED ISSUE #1196: [MTB] allow user to supply a second namespace and tenant admin for PL2 checks
Currently kubectl-mtb allows:

```
kubectl mtb run -n <namespace> --as <tenant-admin>
```

For PL2 tests we need another tenant namespace and admin. We can extend mtb to allows this as:

```
kubectl mtb run -n <namespace> --as <tenant-admin> \
    --other-namespace <namespace2> --other-tenant-admin <tenant-admin2>
```

### PR DESCRIPTION
**Pull request by phoenixking25**
/assign @JimBugwadia @Divya063 @realshuting 

- Refactored the code
- Added other namespace flag and other tenant admin flag
- Added namespaceRequired config in config yaml, to define how many namespace required for benchmark to run
- If there is no additional namespace is given and namespaceRequired is greater than 1, then that benchmark will be skipped

fixes #1196

### CODE REVIEW

File: benchmarks/kubectl-mtb/internal/kubectl-mtb/run.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ package kubectl
 import (
 	"context"
 	"fmt"
+	"sigs.k8s.io/multi-tenancy/benchmarks/kubectl-mtb/test/utils/log"
```
Comment (kishoregv) [inline benchmarks/kubectl-mtb/internal/kubectl-mtb/run.go:20]: I guess this reordering of import log is by accident.

Comment (phoenixking25) [reply benchmarks/kubectl-mtb/internal/kubectl-mtb/run.go:20]: yeah

**Comment by phoenixking25** [comment]
LGTM @JimBugwadia

**Review by JimBugwadia — APPROVED** [review]

**Pull request merged**